### PR TITLE
Docs, screenshots, sort, fixes 

### LINKED ISSUE #91: Design changes proposals
Some suggestions to improve Tagger interface.

- [x] To achieve adaptiveness add bottom bar, move Web Services, Tag Actions and Apply buttons there, and also add a toggle button to control flap.
- [x] Hide rigth pane for empty folders.
- [x] Show StatusPage in right pane when nothing is selected, instead of showing empty tags entries.
- [x] Move advanced search help to yelp docs.
- [x] Avoid using EntryRow for read-only file properties: ActionRow with label in suffix can be used for duration and file size, and for fingerprint we could not show fingerprint but add a button to copy it to clipboard.
- [x] Maybe move search entry to bottom bar? Or at least add separator between it and the files list.
- [x] Improve album art, to show that the user can click to change it. Change cursor on hover, and add little edit icon in the corner of image.

### LINKED ISSUE #122: Update Screenshots

### LINKED ISSUE #125: Add More Sorting Options
Add sort by file path, sort by album, sort by genre

### PR DESCRIPTION
**Pull request by fsobolev**
- [x] Fixed valign of files list
- [x] <s>Made the app non-unique, so a new instance of the app will not affect already running instance</s>
- [x] Updated screenshots and banners
- [x] Docs
  - [x] Advanced search
  - [x] Preferences
  - [x] Web services (I need help with this, because I never used web services and don't quite understand what to write about them in docs)
- [x] Show docs page instead of in-app advanced search help
- [x] More sorting options

Closes #91
Closes #122 
Closes #125

### CODE REVIEW

**Comment by fsobolev** [comment]
> > Made the app non-unique, so a new instance of the app will not affect already running instance
> 
> Didn't look at the code yet (not home currently)...but when a sdcond instance is started did you make it where the previous folder isn't opened (because it would be the same opened in the first instance, no?)

I only changed Gio Application Flags, so yes, a new instance will open whatever was saved in Configuration. But since it's non unique, a new instance will not affect old instance at all, unlike previously when all windows shared the same controller.

**Comment by nlogozzo** [comment]
> I only changed Gio Application Flags, so yes, a new instance will open whatever was saved in Configuration

The problem is with that is if a second instance is opened, whatever is in the Config will always be the folder from the first instance...so if it's a big folder it'll waste time opening the folder just to change it in the second instance.

Is there a way to detect if it's a second/third/whatever that's not the first instance, that way we just won't load from config with those instances.

**Comment by fsobolev** [comment]
> > I only changed Gio Application Flags, so yes, a new instance will open whatever was saved in Configuration
> 
> The problem is with that is if a second instance is opened, whatever is in the Config will always be the folder from the first instance...so if it's a big folder it'll waste time opening the folder just to change it in the second instance.
> 
> Is there a way to detect if it's a second/third/whatever that's not the first instance, that way we just won't load from config with those instances?

I don't think so... What we can do instead is if there's already an instance running then present the window instead of creating a new one.

**Comment by nlogozzo** [comment]
> > > I only changed Gio Application Flags, so yes, a new instance will open whatever was saved in Configuration
> > 
> > The problem is with that is if a second instance is opened, whatever is in the Config will always be the folder from the first instance...so if it's a big folder it'll waste time opening the folder just to change it in the second instance.
> > 
> > Is there a way to detect if it's a second/third/whatever that's not the first instance, that way we just won't load from config with those instances?
> 
> I don't think so... What we can do instead is if there's already an instance running then present the window instead of creating a new one.

I think this would be best if we can't detect multiple instances.

We could maybe add a New Window action in the main window where we can save in config that we are opening a new instance and see that in the new instance and prevent loading the previous folder.

**Comment by fsobolev** [comment]
image

Sorting by album doesn't work well. First, numbers: "10" goes before "9". Second, the the tracks in an album are placed randomly (on screenshot).

**Comment by nlogozzo** [comment]
@fsobolev Could you try the latest commit?

**Comment by fsobolev** [comment]
Still weird sorting by album

**Comment by nlogozzo** [comment]
> Still weird sorting by album

Does it sort them the same way every time though or is it a new random order every time?

**Comment by fsobolev** [comment]
> Does it sort them the same way every time though or is it a new random order every time?

Same way

image

The tracks shown are from 1 album, but they are sorted weirdly, neither by number nor track name nor file name.

File: NickvisionTagger.GNOME/Views/MainWindow.cs [hunk 1]
```diff
@@ -1024,10 +1029,8 @@ private bool SelectedMusicFilesPropertiesChanged()
             }
             else
             {
-                using var loader = GdkPixbuf.PixbufLoader.New();
-                gdk_pixbuf_loader_write(loader.Handle, art.Data, (uint)art.Data.Length, 0);
-                _albumArtImage.SetPixbuf(loader.GetPixbuf());
-                gdk_pixbuf_loader_close(loader.Handle, IntPtr.Zero);
+                var texture = gdk_texture_new_from_bytes(g_bytes_new(art.Data, (uint)art.Data.Length), IntPtr.Zero);
```
Comment (nlogozzo) [inline NickvisionTagger.GNOME/Views/MainWindow.cs:1032]: g_object_unref needs to he called on gdk.texture and g.bytes after the picture is set

**Review by nlogozzo — CHANGES_REQUESTED** [review]

File: NickvisionTagger.GNOME/Views/MainWindow.cs (hunk 1 above)
Comment (fsobolev) [reply NickvisionTagger.GNOME/Views/MainWindow.cs:1038]: GBytes are not GObject, using unref on them causes silent crash.
Added unref on texture.

Comment (nlogozzo) [reply NickvisionTagger.GNOME/Views/MainWindow.cs:1038]: GBytes has g_bytes_unref function to use

**Review by nlogozzo — CHANGES_REQUESTED** [review]

**Review by nlogozzo — APPROVED** [review]

**Pull request merged**